src/lib.rs

```diff
@@ -7,23 +7,17 @@
 //! or for solving bitvector formulas of SMT solvers.
 //! 
 //! Internally `ApInt` uses small-value optimization for values with a bit-width less than or
-//! equal to `64` bits. It uses `64` bit digits and thus its algorithms computes within the base
-//! of 2<sup>64</sup>.
+//! equal to `64` bits. It uses `64` bit `Digit`s (by default, however it can be configured to use
```

Comment:
how? have we implemented this yet? do we want this? I am not sure. it will greatly complicate everything.